### PR TITLE
Migration merge

### PR DESCRIPTION
**Pull request by Da-Scher**
test-mansion mod pack separated into their working minecraft versions. Actions set up.

### CODE REVIEW

File: .github/workflows/on-push.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - name: Set up JDK 1.17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: 17
+      - uses: minepkg/action-crash-test@v0
+        with:
+          no-build: true
+          working-directory: ${{ matrix.directory }} # optional
+          # minecraft: 1.18.2 # optional
```
Comment (fiws) [inline .github/workflows/on-push.yaml]: can we get rid of the comments? They are just instructions in how to use the action.

Also please add a newline at the end

File: 1.16/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.tmp
+minecraft
```
Comment (fiws) [inline 1.16/.gitignore]: missing newline

**Review by fiws — COMMENTED** [review]
lookin good so far. just 2 "mimimi" things.

I'm also wondering how big the repository will be after duplicating all the minecraft maps. Could you check that real quick?
Might have to do some spooky symlinking if it's too big.

**Comment by Da-Scher** [comment]
> lookin good so far. just 2 "mimimi" things.
> 
> I'm also wondering how big the repository will be after duplicating all the minecraft maps. Could you check that real quick? Might have to do some spooky symlinking if it's too big.

~64 MB

File: .github/workflows/on-push.yaml (hunk 1 above)
Comment (Da-Scher) [reply .github/workflows/on-push.yaml]: fixed in 3c3efa3

File: 1.16/.gitignore (hunk 2 above)
Comment (Da-Scher) [reply 1.16/.gitignore]: fixed in 3c3efa3

File: .github/workflows/on-push.yaml (hunk 1 above)
Comment (Da-Scher) [reply .github/workflows/on-push.yaml]: missed that newline bit, fixed in d3f32b7

**Comment by fiws** [comment]
We want the license, readme and gitignore on the top level only.
the gitignore needs to be adjusted to account for the subfolders.

**Comment by Da-Scher** [comment]
Whoops, yeah that makes sense especially with .gitignore. License, readme, and gitignore on top level only. the .gitignore accounts for subdirectories. Also I fixed the github actions so it doesn't run push and PR jobs simultaneously.

**Pull request merged**